### DISCUSSION TITLE
Integrating multiple 10x Multiome (scRNA+ATAC) data-Harmony vs CCA

### DISCUSSION THREAD

**Discussion by bhekmat** [stuart-lab/signac#1562] [Q&A] [answered]
@timoast 

Hi, 

I currently have two (Ctrl and KO) samples of 10x single cell multiome (RNA+ATAC) and am trying to decide the best integration approach. I already tried the CCA approach to find anchors for the SCT assay, and I was wondering if I were to use Harmony on the merged object, should I run Harmony on the SCT assay or both the SCT and LSI? 

Also, do you recommend calling peaks on the integrated assay containing both the Ctrl and KO cells or on each dataset separately? 

Thank you for your help.

**Comment by timoast** [comment, marked as answer]
Hi, I don't think there are any hard-and-fast rules here. I would recommend trying different approaches and evaluating them to see which works best for your dataset.
